### DISCUSSION TITLE
It seems like tanstack router is not yet compatible for phone's builtin browser, am I right?

### DISCUSSION THREAD

**Discussion by szmazhr** [TanStack/router#976] [General]
I am made a web app, after releasing I received feedback (bug report) from some users that website is showing blank page.
after collecting more information from them I found that it is not working in built in browser's like vivo, iQoo, and maybe oppo browser.

After digging into it, found that the main cause is router. This is first time I am using Tanstack Libraries and I don't know much about it. May it could be solve by some configuration.

Below is the code, just a copy from [QuickStart](https://tanstack.com/router/v1/docs/quick-start) page. Which is not working on iqoo browser

code: https://github.com/szmazhr/testing-trbmb
live: https://ghpages.shahzar.in/testing-trbmb/

see the screen recording below

https://github.com/TanStack/router/assets/18204561/c07c012d-58af-4aa0-8d01-01afdc4a506d

**Comment by schiller-manuel** [comment]
can you generate logs/stack traces from those browsers?

**Reply by szmazhr** [reply]
I tried wrapping the whole app is try...catch block. But no chance.
then instead of using npm package i cloned it from github and did some logging to each step link under RouterProvider `router/packages/react-router/src/RouterProvider.tsx`  then Matches `router/packages/react-router/src/Matches.tsx`

## I did this
![image](https://github.com/TanStack/router/assets/18204561/2faa9a24-c87c-48f0-aa12-33e66c2a668c)

## and got this

### in iqoo browser
![WhatsApp Image 2024-01-13 at 19 11 06](https://github.com/TanStack/router/assets/18204561/c3f1cc4a-d2c3-435e-897d-9e306573c90d)

### in chrome
![WhatsApp Image 2024-01-13 at 19 11 05](https://github.com/TanStack/router/assets/18204561/c639b49b-360d-41d2-85b3-3507d80904c2)

Please let me know if there is some way to get the logs.

**Reply by szmazhr** [reply]
Here I got something. I tried visiting the page on Chrome Android 83.0.4103.101 (410410183). Here are the logs.

![Screenshot from 2024-01-13 22-09-33](https://github.com/TanStack/router/assets/18204561/9119d932-d9f2-4a92-8403-bedd3e57cc63)
[192.168.1.4-1705163870163.log](https://github.com/TanStack/router/files/13929162/192.168.1.4-1705163870163.log)
[192.168.1.4-1705163881403.log](https://github.com/TanStack/router/files/13929163/192.168.1.4-1705163881403.log)
[192.168.1.4-1705163888388.log](https://github.com/TanStack/router/files/13929164/192.168.1.4-1705163888388.log)
[192.168.1.4-1705163893161.log](https://github.com/TanStack/router/files/13929165/192.168.1.4-1705163893161.log)

**Reply by schiller-manuel** [reply]
that's the same error as here:

https://github.com/TanStack/router/issues/925

great you could get a stack trace!

**Reply by szmazhr** [reply]
Yeah, you are right. That is the same issue.
Is there anything which I can do for the fix? Or contribute somehow?

**Reply by schiller-manuel** [reply]
I was able to reproduce this issue in an old chrome version. Let's continue in the other issue

**Reply by schiller-manuel** [reply]
fixed in v1.4.3
